src/Sarif/CodeGenHints.json

```diff
@@ -50,6 +50,23 @@
       "kind": "DictionaryHint"
     }
   ],
+  "Rule.DefaultLevel": [
```

Comment:
Yes, we need the same Schema feature -- the ability to specify that a defined type in the schema is an enum rather than a class -- to properly support SuppressionState (because that's what would allows us to define an array of SuppressionState). I'll do it this week.